app/Console/Commands/SetmoreAppointments.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\SetmoreAppointment;
+use Carbon\Carbon;
+use Config;
+use DB;
+use GuzzleHttp\Client;
+use Illuminate\Console\Command;
+
+class SetmoreAppointments extends Command {
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'SetmoreAppointments:Update';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Fetches training appointments from Setmore.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct() {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle() {
+        $setmoreAccessToken = null;
+        $setmoreAppointments = $setmoreServices = $setmoreStaff = [];
+        $from = Carbon::now()->subHours(2)->toDateString();
+        $to = Carbon::now()->addDays(90)->toDateString();
+        $client = new Client();
+        try {
+            $res = $client->request('GET', Config::get('setmore.endpoint') . '/o/oauth2/token?refreshToken=' . Config::get('setmore.api_key'));
+            $setmoreAuthResp = (string) $res->getBody();
+            $setmoreAuthRespA = json_decode($setmoreAuthResp, true);
+        } catch (ClientErrorResponseException $exception) {
+            echo "Unable to fetch access token from Setmore API: $exception";
+        }
+        if (isset($setmoreAuthRespA) && isset($setmoreAuthRespA['response'])) {
+            if ($setmoreAuthRespA['response'] && isset($setmoreAuthRespA['data']['token'])) {
+                $setmoreAccessToken = $setmoreAuthRespA['data']['token']['access_token'];
+            }
```

Comment:
In this catch block, we can return 1 to show that there was an error and keep from moving on. Otherwise we don't want to continue. This should also allow us to clean up this if statement a little bit (if it's even necessary to leave here)

Reply:
Yeah, I see what you mean. I've cleaned this up a bit and consolidated the token variable assignment in the try structure.